### DISCUSSION TITLE
Process community rewards cycle: 4/30 -> 5/27

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#119] [Rewards]
It escaped me that another month has passed, and we are now one week past the end date of our current 4-week rewards cycle, which spans from 4/30 -> 5/27.

Based on our switch to Github-based discussions, I have written a [rough outline of the process ](https://github.com/asastats/docs/blob/main/rewards-process.md) I used to calculate the last rewards cycle (also archiving related Github issues), which can be refined further as others participate in the processing.

Anyone else want to try out this task?  @Babbexx22, @SCN9A, @bear1bear2bear3, @rom1henry?

I can of course fill in any blanks to help along the way as needed.

**Comment by ipaleka** [comment]
If we'd be creating project's historical NFTs, then one is reserved for the scene where a teacher in front of a class asks for volunteers to solve an equation in front of a whiteboard, and pupils move their eyesight to not be crossed with the teacher's. We'll call it TeacheRhythMatic. :)

**Reply by wxivd** [reply]
> Anyone else want to try out this task?

Not myself, thanks :satisfied:
I appreciate the great work @AlgoRhythMatic is doing. I am happy for him to lead the way.

**Comment by AlgoRhythMatic** [comment]
Ok, at this point I'm taking silence as confirmation that there is no specific interest this round, and will proceed to process the rewards per current guide steps.  I will ask again next month (ideally ahead of schedule to allow for more community eyes to see or raise their hands).

**Comment by AlgoRhythMatic** [comment]
Rewards output for review per measurements listed [here in the spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857)

``` 
# NOTE ASA Stats Community rewards 4/30 - 5/27
AlgoRhythMatic 0.08 damo
DragMZ 0.04 damo
iMpacToFTrees 0.02 damo
kerrilija 0.08 damo
moloch10  0.16 damo
motuwagon 0.02 damo
rach 0.02 damo
SCN9A 0.16 damo
StonedAlgo 0.02 damo
Tink 0.02 damo
ym982 0.1 damo
TOTAL 0.72 damo
```

**Reply by AlgoRhythMatic** [reply]
Now that ~24 hrs has passed w/ no commentary, I think that the list above can be considered final.

@ipaleka - please the list above per your convenience!
